examples/time-widget.html

```diff
@@ -0,0 +1,402 @@
+                          "layer_name": "Árboles",
+                          "cartocss": "/** simple visualization */\n\n#arboles{\n  marker-fill-opacity: 0.9;\n  marker-line-color: #FFF;\n  marker-line-width: 1;\n  marker-line-opacity: 1;\n  marker-placement: point;\n  marker-type: ellipse;\n  marker-width: 10;\n  marker-fill: green;\n  marker-allow-overlap: true;\n}",
+                          "cartocss_version": "2.1.1",
+                          "interactivity": "cartodb_id",
+                          "table_name": "\"\".",
+                        },
+                        "widgets": {
+                        //   'category_widget_uuid': {
+                        //     "type": "aggregation",
+                        //       "title": "Categorías de árboles",
+                        //       "column": "name",
+                        //       "aggregation": "count",
+                        //       "sync": true
+                        //   }
+                          "list_widget": {
+                            "type": "list",
+                            "title": "Arboles",
+                            "bbox": false,
+                            "columns": [
+                              "cartodb_id",
+                              "name"
+                            ],
+                            "columns_title": [
+                              "id",
+                              "tipo"
+                            ]
+                          },
+                        }
+                      },
+                      {
+                        "id": "101e17db-1466-4842-8eef-f81da32f48db",
+                        "type": "CartoDB",
+                        "infowindow": {},
+                        "tooltip": {},
+                        "legend": {},
+                        "order": 3,
+                        "visible": true,
+                        "options": {
+                          // TODO extract is a hack, until Winshaft can handle a Date column
+                          "sql": "select *, extract(epoch from end_at) AS numeric_end_at from mwc13",
+                          "layer_name": "mwc13-static",
+                          "cartocss": "/** simple visualization */\n\n#mwc13{\n  marker-fill-opacity: 0.9;\n  marker-line-color: #FFF;\n  marker-line-width: 1;\n  marker-line-opacity: 1;\n  marker-placement: point;\n  marker-type: ellipse;\n  marker-width: 10;\n  marker-fill: #229A00;\n  marker-allow-overlap: true;\n  marker-comp-op: multiply;\n}",
+                          "cartocss_version": "2.1.1",
+                          "interactivity": "cartodb_id",
+                          "table_name": "mwc13"
+                        },
+                        "widgets": {
+                          "time_static_widget_uuid": {
+                            "type": "histogram",
+                            column_type: 'date', // to differentiate from a quantities histogram
```

Comment:
The thing is that the models and views are created at the time of parsing the vizjson. I'm not very keen to complicate the view creation right now if we can avoid it.

E.g. for a torque layer I can just check the layer type to take the decision. The problem right now is that the layer type is the same as the existing cartodb layer type so can't distinguish them. Other ideas?